### PR TITLE
Apim 6853 consol setting both portals

### PR DESCRIPTION
**Pull request by illichso**
## Issue

https://gravitee.atlassian.net/browse/APIM-6853

## Description

The warning message "Apply to Both Portals" should also be displayed for API Key Header field in Portal->General section

Screenshot 2024-09-17 at 15 23 11

https://github.com/user-attachments/assets/f2861613-bb59-4cce-b730-29ad5bd6baa1

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-bhryftroim.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.html [hunk 1]
```diff
@@ -283,7 +290,7 @@ <h3 class="portal__form__card__title">Options</h3>
           [matTooltipDisabled]="!isReadonly('portal.analytics.enabled')"
           class="portal__form__card__form-field"
         >
-          <gio-form-label>Add Google Analytics</gio-form-label>
+          <gio-form-label> Add Google Analytics</gio-form-label>
```
Comment (phiz71) [inline gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.html]: Remove the space

File: gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.spec.ts [hunk 2]
```diff
@@ -320,6 +324,33 @@ describe('PortalSettingsComponent', () => {
...
+    it('applies to both portals is present', async () => {
+      const appliesToBothPortals = 'Applies to both portals';
+      portalSettingsMock = fakePortalSettings({
+        portalNext: {
+          access: { enabled: true },
+        },
+      });
+      expectPortalSettingsGetRequest(portalSettingsMock);
+      expect(await componentHarness.getBadgeWarningText()).toEqual(appliesToBothPortals);
+      expect(await componentHarness.getBadgeIconName()).toBeTruthy();
```
Comment (phiz71) [inline gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.spec.ts]: I don't think this "expect" is relevant.
By checking the icon, I would expect the name of the icon to be the right one.
Here we are just verifying that there is an icon.

Please could you either remove the expectation on the badge icon **or** check that the icon is the right one (by checking the name for instance).

File: gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.html (hunk 1 above)
Comment (illichso) [reply gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.html]: Done

File: gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.spec.ts (hunk 2 above)
Comment (illichso) [reply gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.spec.ts]: updated

**Review by phiz71 — APPROVED** [review]

**Pull request merged**